openwisp_users/admin.py

```diff
@@ -106,13 +106,15 @@ def get_formset(self, request, obj=None, **kwargs):
         """
         formset = super().get_formset(request, obj=obj, **kwargs)
         if request.user.is_superuser:
+            self.select_organization(request, formset.form)
             return formset
         if not request.user.is_superuser:
             formset.form.base_fields[
                 'organization'
             ].queryset = Organization.objects.filter(
                 pk__in=request.user.organizations_managed
             )
+        self.select_organization(request, formset.form)
         return formset
```

Comment:
Can we write it like this?
```suggestion
        if not request.user.is_superuser:
            formset.form.base_fields[
                'organization'
            ].queryset = Organization.objects.filter(
                pk__in=request.user.organizations_managed
            )
        self.select_organization(request, formset.form)
        return formset
```